tfjs-backend-webgpu/src/base.ts

```diff
@@ -57,7 +57,12 @@ if (isWebGPUSupported()) {
     };
 
     const device: GPUDevice = await adapter.requestDevice(deviceDescriptor);
-    const adapterInfo = await adapter.requestAdapterInfo();
+    const adapterInfo =
+      'info' in adapter
+        ? adapter.info
+        : 'requestAdapterInfo' in adapter
+          ? await (adapter as any).requestAdapterInfo()
```

Comment:
You'll need `/* tslint:disable-next-line:no-any */` right before this, or it may be better to cast `(adapter as unknown as GPUAdapterInfo)` instead.
```suggestion
          ? await (adapter as unknown as GPUAdapterInfo).requestAdapterInfo()
```

Reply:
Actually, this is wrong, my bad. The promise result should be `GPUAdapterInfo`. Just adding the `/* tslint:disable-next-line:no-any */` should fix it.